Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -9,6 +9,8 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
     private var site: Site
     private var actions: [ActionOverlayTableViewAction]
     private var tableView = UITableView()
+    private var headerIconHandler: () -> UIImage?
```

Comment:
So instead of defining these as closures that return an image. You can just make it `siteImage` and make it a UIImage. same goes with backgroundColor. Just make it `private var siteBackgroundColor: UIColor?`